src/picker/fzf.rs

```diff
@@ -0,0 +1,76 @@
+use crate::error;
+use crate::models;
+use crate::picker;
+use error::PickerError;
+use itertools::Itertools;
+use models::ResultWithDefaultError;
+use picker::{ItemPicker, PickableItem};
+use std::collections::HashMap;
+use std::io;
+use std::io::Write;
+use std::process::{Command, Stdio};
+
+pub struct FzfPicker;
+
+fn remove_trailing_newline(value: String) -> String {
+    let mut chars = value.chars();
+    chars.next_back();
+    chars.as_str().to_string()
+}
+
+fn format_as_fzf_input(items: &[PickableItem]) -> String {
+    items
+        .iter()
+        .map(|item| item.formatted.clone())
+        .unique()
+        .fold("".to_string(), |acc, item| acc + item.as_str() + "\n")
+}
+
+fn create_element_hash_map(items: &[PickableItem]) -> HashMap<String, i64> {
+    items
+        .iter()
+        .map(|item| (item.formatted.clone(), item.id))
+        .collect::<HashMap<String, i64>>()
```

Comment:
This would have collisions for time-entries that format to the same string value.
Just thinking out of loud if that is a problem in case of continuing a time entry.

Reply:
But in our code, currently we only use the description and duration as parts of the string output, do you think you using selected entry's index in list would be a better option?

Reply:
Maybe we could inject the index / time-entry id into the fzf input as part of `format_as_fzf_input`

Reply:
> This would have collisions for time-entries that format to the same string value

Good catch! I removed the `unique` inside `format_as_fzf_input` and now this behaves as it should. We still need to make sure that two time entries only ever print the same thing if they are exactly the same. This basically means:

- Showing project name and Id
- Showing task name and id
- Show all tags

Though I think we should solve this in a different issue, if you agree

Reply:
Agreed, it is an edge case 👍